### PR TITLE
add credits

### PR DESCRIPTION
**Pull request by AvinaashGhansam**

### CODE REVIEW

File: src/Information/InformationModal.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}) => {
+  const [isAccordionOpen, setIsAccordionOpen] = useState<number | null>(null);
+
+  const handleAccordionClick = (index: number) => {
+    if (isAccordionOpen === index) {
+      setIsAccordionOpen(null);
+    } else {
+      setIsAccordionOpen(index);
+    }
+  };
```
Comment (zlix12) [inline src/Information/InformationModal.tsx:30]: Why are we managing the open item? Looking at the chakra docs it should manage the open item on its own

File: src/Information/InformationItem.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          <Text>
+            This application is developed by:
+            <OrderedList>
+              <ListItem>Ana Diru</ListItem>
+              <ListItem>Anthony Zigerelli</ListItem>
+              <ListItem>Avinaash Ghansam</ListItem>
+              <ListItem>David Casey</ListItem>
+            </OrderedList>
+          </Text>
+        )}
```
Comment (zlix12) [inline src/Information/InformationItem.tsx:50]: Let's just take the accordion content as `children: React.ReactNode`. Right now this is pretty coupled to the information.

Comment (zlix12) [reply src/Information/InformationItem.tsx:50]: To clarify, the parent should provide this content through a children prop

File: src/Information/InformationItem.tsx
Comment (zlix12) [inline src/Information/InformationItem.tsx:1]: Can you change this to `src/information/InformationItem`? (Capitalization of the folder)

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: src/Information/InformationItem.tsx
Comment (AvinaashGhansam) [reply src/Information/InformationItem.tsx:1]: changed made

File: src/Information/InformationItem.tsx (hunk 2 above)
Comment (AvinaashGhansam) [reply src/Information/InformationItem.tsx:50]: change made to reflect this

File: src/Information/InformationModal.tsx (hunk 1 above)
Comment (AvinaashGhansam) [reply src/Information/InformationModal.tsx:30]: fixed?

**Comment by zlix12** [comment]
@AvinaashGhansam we should add the disclaimer to this. From the Nuts & Bolts channel.

**Review by zlix12 — DISMISSED** [review]
LGTM

**Review by ajd6735 — APPROVED** [review]
Looks great

**Pull request merged**